package.json

```diff
@@ -17,7 +17,7 @@
         "check:formatting": "prettier --check .",
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
-        "clean": "ts-node ./script/clean.ts && npm run clean --workspaces --if-present",
+        "clean": "ts-node ./script/clean.ts && npm run clean --workspaces --if-present && sh ./script/git-clean.sh",
```

Comment:
if git will remove anything not under source control, do we need to run `ts-node ./script/clean.ts && npm run clean --workspaces --if-present` first?

Reply:
I think keeping it this way, makes it easier to see the files that `git clean` would delete, since it prompts a much shorter list of files, potentally avoiding unintended deletes (newly created files currently unstaged), hence making it safer for the user. With a long list of mostly all modules files the user might assume its all modules imo.